.github/actions/opensearch/run.sh

```diff
@@ -42,17 +42,32 @@ do
     opensearchproject/opensearch:${STACK_VERSION}
 done
 
-docker run \
-  --network cluster \
-  --rm \
-  appropriate/curl \
-  --max-time 120 \
-  --retry 120 \
-  --retry-delay 1 \
-  --retry-connrefused \
-  --show-error \
-  --silent \
-  http://os1:$PORT
+if [[ $DISABLE_SECURITY = true ]]; then
```

Comment:
Looks like we can refactor by only building arguments based on security status

Reply:
I feel it's clearner to separate out the complete run command rather than adding checks on arguments.